docs/installing/manual_install.md

```diff
@@ -514,6 +521,33 @@ useful to you. Change the variables to suit your installation.
     chown root:alaveteli /etc/init.d/alaveteli
     chmod 754 /etc/init.d/alaveteli
 
+**Set up the cluster (optional):**
```

Comment:
This section can be removed, as we'll be making the `--server` flag available to the rake task.